core/includes/bootstrap.inc

```diff
@@ -121,49 +121,54 @@ define('WATCHDOG_DEPRECATED', 8);
  */
 
 /**
- * First bootstrap phase: initialize configuration.
+ * First bootstrap phase: initialize database layer.
  */
-define('BACKDROP_BOOTSTRAP_CONFIGURATION', 0);
+define('BACKDROP_BOOTSTRAP_DATABASE_EARLY', 0);
```

Comment:
Rather than adding a new bootstrap phase, would it be possible to just make the config bootstrap phase "upgrade" the boostrap level in the contructor for the new database config storage class? This is the same thing that `BackdropDatabaseCache::__contstruct()` does. Even though the cache backend is needed at boostrap level 2, it upgraded the level to 3 so that it can serve the page cache from the database.